packages/hs-react-ui/src/components/Button/Button.stories.tsx

```diff
@@ -16,7 +20,7 @@ export default {
 
 export const Basic = () => (
   <Button
-    buttonType={select('Button Type', ButtonTypes, ButtonTypes.primary)}
+    StyledContainer={ButtonContainers[select('Button Container', ButtonTypes, ButtonTypes.primary)]}
```

Comment:
The label on the knob here should reflect the name of the prop exactly "Styled Container" or "StyledContainer